common/lib/xmodule/xmodule/video_module/video_xfields.py

```diff
@@ -173,3 +173,12 @@ class VideoFields(object):
         scope=Scope.settings,
         default="",
     )
+    date_last_view_bumper = DateTime(
+        display_name=_("Date of the last view of the bumper"),
+        scope=Scope.preferences,
+    )
+    do_not_show_again_bumper = Boolean(
```

Comment:
Nit: I would have gone with `bumper_last_view_date` and made the later `bumber_show_again` (aren't negatives in variable names hard to parse?). :)